src/Console/Command/FixCommandHelp.php

```diff
@@ -79,6 +79,7 @@ public static function getHelpCopy()
 
 A combination of <comment>--dry-run</comment> and <comment>--diff</comment> will
 display a summary of proposed fixes, leaving your files unchanged.
+Optionally the <comment>--diff</comment> can be used to specify the diff. output format; ``--diff diff`` (default) or ``--diff diff-short``.
```

Comment:
`SebastianBergmannDiff` is an internal detail of how diff is implemented, this should not be exposed to end user IMO. Plus, neither `SebastianBergmannDiff` nor `sbd` tells me what the actual result will look like.